src/routes/blogs.js

```diff
@@ -7,6 +7,18 @@ router.route("/blogs").get((req, res) => {
   posts.allPosts.then(data => res.render("blogs", data));
 });
 
+router
+  .route("/blogs/create")
+  .get((req, res) => {
+    res.render("blogform");
+  })
+  .post((req, res) => {
+    posts.newPosts(req.body).then(newBlogpost => {
+      res.setHeader("Content-Type", "application/json");
+      res.json(newBlogpost);
```

Comment:
Since it's a user who is writing a post, and not a programmer using our API, it would be more user-friendly to redirect after blog post creation to either: 
* The created post page
* All blog posts page